pyasic/miners/backends/epic.py

```diff
@@ -114,6 +115,52 @@ async def get_config(self) -> MinerConfig:
         self.config = cfg
         return self.config
 
+    async def send_config(self, config: MinerConfig, user_suffix: str = None) -> None:
+        self.config = config
+        conf = self.config.as_epic(user_suffix=user_suffix)
+
+        try:
+            # Temps
+            data = await self.web.set_shutdown_temp(conf["temp_control"]["hot_temp"])
+            # Fans
+            if conf["fan_control"]["mode"] == "Manual":
+                data = await self.web.set_fan(
+                    {"Manual": int(conf["fan_control"]["speed"])}
+                )
+            if conf["fan_control"]["mode"] == "Auto":
+                data = await self.web.set_fan(
+                    {
+                        "Auto": {
+                            "Idle Speed": int(conf["fan_control"]["speed"]),
+                            "Target Temperature": int(
+                                conf["temp_control"]["target_temp"]
+                            ),
+                        }
+                    }
+                )
```

Comment:
Ideally the fan_mode config would just return this dictionary already parsed, so you can just use it directly.

Reply:
I get that this is tough because of the use of multiple values, but thats what the merge function is for, see https://github.com/UpstreamData/pyasic/blob/f21fda5e388afaca934125dbd3e2f05fe19bef6f/pyasic/config/__init__.py#L183-L191